code/modules/clothing/spacesuits/rig/suits/combat.dm

```diff
@@ -48,11 +45,6 @@
 	offline_slowdown = 3
 	offline_vision_restriction = 0
 
-	helm_type = /obj/item/clothing/head/helmet/space/rig/ihs_combat
```

Comment:
Was this intended deletion of `helm_type`?

Reply:
not intended